lib/realtime/vehicle.ex

```diff
@@ -189,6 +192,7 @@ defmodule Realtime.Vehicle do
       via_variant: via_variant,
       bearing: VehiclePosition.bearing(vehicle_position),
       block_id: block_id,
+      block_id_with_overload: block_id_with_overload,
```

Comment:
It looks aside from the off course check, this value isn't used anymore, so we could take the field out of the struct.

Reply:
At least for the moment I want to leave it on the client so we can debug the data in the browser without having to add breakpoints in the server or whatever.

Reply:
I don't like the idea of adding a field to the vehicle struct just for debugging. What about making a `data_discrepancy` for `block_id`? (Then it'd also show up when the debug flag is on.)

Reply:
Ok, changed to that.